linkedlistmiddleelement.java

```diff
@@ -0,0 +1,71 @@
+class LinkedList 
+{ 
+    Node head; // head of linked list 
+  
+    /* Linked list node */
+    class Node 
+    { 
+        int data; 
+        Node next; 
+        Node(int d) 
+        { 
+            data = d; 
+            next = null; 
+        } 
+    } 
+  
+    /* Function to print middle of linked list */
+    void printMiddle() 
+    { 
+        Node slow_ptr = head; 
```

Comment:
int count=0;
        Node slow_ptr=head;
        Node fast_ptr=head;
        while(slow_ptr!=null)
        {
        count++;
        slow_ptr=slow_ptr.next;
        }
        for(int i=0;i<=count/2;i++)
        {
        if(i==count/2)
        {
        System.out.println(fast_ptr.data);
        }
        fast_ptr=fast_ptr.next;
        }